### ISSUE TITLE
Erzya is ready to be launched!

### ISSUE THREAD

**Issue by ftyers** [common-voice/common-voice#3254]
Over 5000 sentences and [90% translated](https://pontoon.mozilla.org/myv/common-voice/) :)

**Comment by MichaelKohler** [comment]
Indeed, that fact got correctly recognized with the last export from Sentence Collector, and it will go live with the next website deployment: https://github.com/common-voice/common-voice/commit/eb5d8bab793c9168e076d1a2cebb171b8c16c6cd 🎉

**Comment by ftyers** [comment]
Something went wrong in the release, would it be possible to get Erzya enabled? There is going to be a conference in a week or so on Uralic language technology and we'd like to showcase it there :)
